integration_test.go

```diff
@@ -84,6 +84,11 @@ func TestWrapUnwrap(t *testing.T) {
 	assert.NoError(err)
 	t.Logf("CRK created successfully: id=%s\n", crk.ID)
 
+	metadata, err := c.GetKeyMetadata(ctx, crk.ID)
```

Comment:
So I just realized that this isn't really a valid test for Payload since the CreateKey above is creating a root key which will never return payload even on a GetKey.

Probably should be creating a new exportable key to test with the GetKeyMetadata() function to verify it doesn't return payload in that case (it will on GetKey with an exportable)

If you create a new exportable key with the same name `kptest-crk` it will get cleaned up below, but maybe add another if conditional to look for `kptest-exportable` and use that as the name to avoid confusion

Reply:
Good catch, I think we can just make the existing `kptest-crk` key extractable.
```Go
crk, err := c.CreateKey(ctx, "kptest-crk", nil, true)
```

Reply:
As we talked offline, we can't just make the existing key extractable because we won't be able to wrap/unwrap the DEK.
I created a new TestExtractableKey for integration to cover this case.